tests/Common/BranchComponentTest.php

```diff
@@ -1188,20 +1188,34 @@ public function testComponentConfigRowVersionRollback()
             ->setChangeDescription(null);
         $configuration3 = $componentsApi->updateConfigurationRow($configurationRow);
 
-        try {
-            // rollback to version 2
-            $configuration4 = $componentsApi->rollbackConfigurationRow(
-                'wr-db',
-                'main-1',
-                $configurationRow->getRowId(),
-                2
-            );
-            $this->fail('Configuration row rollback should not be implemented');
-        } catch (ClientException $e) {
-            $this->assertSame(501, $e->getCode());
-            $this->assertSame('notImplemented', $e->getStringCode());
-            $this->assertContains('Not implemented', $e->getMessage());
-        }
+        // rollback to version 2
```

Comment:
Este by som otestoval asi `$componentsApi->getConfigurationRow()` ze vrati to co ma

Reply:
Ještě sem sa na to podíval čerstvíma očima, přejmenoval sem ty proměnné, nedávalo to smysl, protože to testuje row a píše configuration, prostě sem udělal předím copy&paste :( 
Přidal sem taky asserty na to jestli je správná verze konfigurace.
Přidal sem assert na row, které vytáhnu znova z DB.